slite.el

```diff
@@ -1,9 +1,58 @@
-;; -*- lexical-binding: t -*-
+;;; slite.el --- Interactively runs your Common Lisp tests  -*- lexical-binding: t -*-
 
-(provide 'slite)
+;; Copyright (C) 2021-2023 Arnold Noronha
+
+;; Author: Arnold Noronha <arnold@tdrhq.com>
+;; Homepage: https://github.com/tdrhq/slite
+;; Keywords: lisp tools
+;; Package-Requires: ((emacs "25.1"))
+;; SPDX-License-Identifier: Apache-2.0
+
+;;; Commentary:
+
+;; Slite stands for SLIme TEst runner. Slite interactively runs
+;; your Common Lisp tests (currently only FiveAM and Parachute are
+;; supported).  It allows you to see the summary of test failures,
+;; jump to test definitions, rerun tests with debugger all from
+;; inside Emacs.
+
+;; You might want to add some key bindings in various Lisp mode maps:
+;;
+;;     (define-key emacs-lisp-mode-map (kbd "C-c v") #'slite-run)
+;;     (define-key lisp-mode-map (kbd "C-c v") #'slite-run)
+;;     (define-key lisp-mode-map (kbd "C-c j")
+;;       #'slite-compile-defun-and-run-tests)
+;;     (with-eval-after-load 'slime
+;;       (define-key slime-mode-map (kbd "C-c v") #'slite-run))
+;;     (with-eval-after-load 'sly
+;;       (define-key sly-mode-map (kbd "C-c v") #'slite-run))
```

Comment:
Could we not add this back as defaults? The lisp-mode-maps are particularly critical. Perhaps a slite-lisp-minor-mode?

Reply:
> Could we not add this back as defaults? The lisp-mode-maps are particularly critical.

I wouldn't do it by default and wouldn't say the bindings are *critical*. The users can initially use `M-x slite- TAB` to find the entry point, and can then proceed to adding a key binding themselves, or look at the documentation to learn more. (So you should probably add a note to the readme, not just the library commentary as I have done.)

> Perhaps a slite-lisp-minor-mode?

I suggested something along those lines in the commit message, but since you modify the keymaps of multiple major modes, you would end up having to define multiple minor modes.

You might want to consider adding a function that adds all those bindings, using your preferred keys. That way you have to instruct users to call that one function in their init file, *iff* they are okay with the keys you have chosen. You could also tell them that if they want to use different keys, they should copy the body of that function to their init file and adjust accordingly.